packages/main/test/samples/Calendar.sample.html

```diff
@@ -0,0 +1,105 @@
+<header>
+	<h2 class="control-header">Calendar</h2>
```

Comment:
Use this header to display the since info
```html
<header class="component-heading">
	<h2 class="control-header">Calendar</h2>
	<div class="component-heading-since">
		<span><!--since_tag_marker--></span>
	</div>
</header>
```

Reply:
Done